tests/python/upgrade_3rdParty.sh

```diff
@@ -0,0 +1,15 @@
+rm -rf 3rdParty 
+pip install -r requirements.txt -t 3rdParty
+git add 3rdParty
+python - <<EOF
+
+import sys
+sys.path.insert(0, '3rdParty')
```

Comment:
this is a bit nitpicky but could we say third_party instead of 3rdparty? (also for the name of this script?)